package.json

```diff
@@ -98,6 +98,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.2.5",
+    "nodemon": "^1.9.1",
```

Comment:
@oliviertassinari my guess is that he added it since the test suite uses it for watching.

However, I specifically left it out of the `package.json` because a lot of users have it globally installed (I think it even throws up a "prefer global install" message when you add it locally)